SPECS/kernel/kernel.spec

```diff
@@ -433,10 +433,12 @@ ln -sf linux-%{uname_r}.cfg /boot/mariner.cfg
 - Add cpupower.service to kernel-tools
 - Enable user-based event tracing
 - Cherry-pick 2.0 kernel config commits
--  * Tue Nov 28 2023 Thien Trung Vuong <tvuong@microsoft.com>
--  - Enable CONFIG_BPF_LSM
--  * Tue Nov 28 2023 Juan Camposeco <juanarturoc@microsoft.com>
--  - Enable CUSE module
+  * Tue Nov 28 2023 Thien Trung Vuong <tvuong@microsoft.com>
```

Comment:
I don't believe this is valid `%changelog` syntax 
https://rpm-packaging-guide.github.io/#working-with-spec-files

Reply:
"Each item begins with a - character."

Reply:
Adjusted each commit's changelog entry to better reflect expected format